pkg/cloudprovider/events/events.go

```diff
@@ -38,3 +42,13 @@ func NodeClaimFailedToResolveNodeClass(nodeClaim *v1beta1.NodeClaim) events.Even
 		DedupeValues:   []string{string(nodeClaim.UID)},
 	}
 }
+
+func FilteredInstanceTypes(nodeClaim *v1beta1.NodeClaim, filteredITsRequirement int, filteredExoticITs int, filteredExpensiveSpot int, filteredITs []string) events.Event {
```

Comment:
Musing: Consider storing all of this information together in a struct rather than making this info into separate arguments to the function and stored in different variables.